modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/GranuleDescriptor.java

```diff
@@ -1060,15 +1061,13 @@ public GranuleLoadingResult loadRaster(
                 granuleURLUpdated = ovrProvider.getOvrURL();
                 assert ovrProvider.getExternalOverviewInputStreamSpi() != null
                         : "no cachedStreamSPI available for external overview!";
-                inStream =
+                SourceSPIProvider sourceSpiProvider =
```

Comment:
I'm not familiar with this bit of code, but the change gives me the impression that before the code allowed to have two different types of format for image and ovr (which is a thing, I believe that GDAL handles jpeg as main image but tiff as ovr for example). While the updated code assumes they are the same?

Reply:
Fixed.